tortoise/threshold.go

```diff
@@ -98,7 +98,7 @@ func computeThresholdForLayers(config Config, epochWeight map[types.EpochID]weig
 	return threshold
 }
 
-func getConfidenceWindow(config Config, state *commonState, tmode mode) types.LayerID {
+func getVerificationWindow(config Config, state *commonState, tmode mode) types.LayerID {
```

Comment:
i think it'll be beneficial to write comments about why this window is important, as discussed in the research call.

Reply:
Not related to this pr though